esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -1795,6 +1823,7 @@ void CWsDfuEx::getFilePartsOnClusters(IEspContext &context, const char* clusterR
             if (clusterInfo) //Should be valid. But, check it just in case.
             {
                 partsOnCluster->setReplicate(clusterInfo->queryPartDiskMapping().isReplicated());
+                partsOnCluster->setCanReplicate(canFileReplicate(clusterName));
```

Comment:
The environment topology changes very, very rarely (and when it does the system is usually restarted).